src/public/app/widgets/containers/split_note_container.js

```diff
@@ -74,6 +74,47 @@ export default class SplitNoteContainer extends FlexContainer {
         appContext.tabManager.removeNoteContext(ntxId);
     }
 
+    async moveThisNoteSplitCommand({ntxId, isMovingLeft}) {
+        if (!ntxId) {
+            logError("empty ntxId!");
+            return;
+        }
+
+        const contexts = appContext.tabManager.noteContexts;
+
+        const currentIndex = contexts.findIndex(c => c.ntxId === ntxId);
+        const leftIndex = isMovingLeft ? currentIndex - 1 : currentIndex;
+
+        if (currentIndex === -1 || leftIndex < 0 || leftIndex + 1 >= contexts.length) {
+            logError("invalid context!");
```

Comment:
It would be useful to dump the `currentIndex`, `leftIndex` and `contexts.length` to the error message for debugging.